test/integration/converter.cc

```diff
@@ -151,3 +151,125 @@ void ParserStringConverter(const std::string &_version)
   EXPECT_EQ(_version, magElem->OriginalVersion());
 }
 
+/////////////////////////////////////////////////
+/// Test conversion unflattened world in 1.7 to 1.8
+TEST(ConverterIntegration, UnflattenConversion)
+{
+  const std::string filename =
+  sdf::filesystem::append(PROJECT_SOURCE_PATH, "test", "sdf",
```

Comment:
Can you change this to `sdf::testing::TestFile`?

Reply:
3e0cd5d70fce3d7dbe32746c2f3ce526ab059baa